components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -13,26 +17,25 @@ import CloudAnnouncementBar from './cloud_announcement_bar';
 import PaymentAnnouncementBar from './payment_announcement_bar';
 import CloudTrialAnnouncementBar from './cloud_trial_announcement_bar';
 
-export default class AnnouncementBarController extends React.PureComponent {
-    static propTypes = {
-        license: PropTypes.object,
-        config: PropTypes.object,
-        user: PropTypes.shape({
-            email: PropTypes.string.isRequired,
-            email_verified: PropTypes.bool,
-        }),
-        canViewSystemErrors: PropTypes.bool.isRequired,
-        latestError: PropTypes.object,
-        totalUsers: PropTypes.number,
-        warnMetricsStatus: PropTypes.object,
-        actions: PropTypes.shape({
-            dismissError: PropTypes.func.isRequired,
-        }).isRequired,
-    }
+type Props = {
+    license?: ClientLicense;
+    config?: Partial<ClientConfig>;
+    user?: Partial<UserProfile>;
```

Comment:
I'm not sure why the propTypes had this as a shape object. Perhaps we could just use UserProfile for this now instead of the partial? If that doesn't work because that the tests don't provide all the fields to make up a user object, you can use `TestHelper.getUserMock` to add the missing ones.

Reply:
Actually, this isn't used, so could you just remove it?

Reply:
there were the same props in `configuration_bar.tsx` I removed these props from that component too. 

also, I removed these props from `announcement_bar/index.ts`